lib/passwordless/router_helpers.rb

```diff
@@ -1,7 +1,14 @@
 # frozen_string_literal: true
 
 module Passwordless
+  # Some helpers for generating passwordless routes.
   module RouterHelpers
+    # Generates passwordless routes for any Model connected to Passwordless.
+    # @param resource [Symbol] the pluralized symbol of a Model (e.g - :users).
+    # @param at [string] Optional - provide custom path (i.e controller)
```

Comment:
@mikker I also wasn't sure if I understood what could be provided for `at` and `as`.
Can you review these lines as well?

Reply:
Perhaps it's best shown with examples:

```
passwordless_for :users, at: 'session_stuff', as: :user_session_things
```

The passwordless engine get's mounted at `at`, eg your urls end up like:

```
/session_stuff/sign_in
```

Your url helpers get scoped under `as`, eg in a view:

```erb
<%= link_to 'Sign in', user_session_things.sign_in_path %>
```